keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -88,7 +88,8 @@ public class KeeperPlugin : Plugin<Project> {
 
   internal companion object {
     const val INTERMEDIATES_DIR = "intermediates/keeper"
-    const val DEFAULT_R8_VERSION = "1.6.53"
+    const val PRINTUSES_DEFAULT_VERSION = "2.2.41"
```

Comment:
Do we need to raise the printuses version?

Reply:
I just checked and now that we're using a different version depending on the entry-point, it should not be a problem.
But use the latest stable version of R8 seems a good idea, no?

Reply:
We can raise it to whatever version ships with studio 4.1, but I want to keep it pinned to the minimum version of studio that we ship with

Reply:
Will be undone with b68f65a